front/src/interfaces/utils/api/IChannel.ts

```diff
@@ -0,0 +1,7 @@
+export default interface ChannelType {
+  isPrivate: boolean;
+  channelName: string;
+  joinCnt: number;
+  totalCnt: number;
+  adminUserIds: number[];
+}
```

Comment:
채널을 만든사람을 operator 라고 하고, operator가 지정한 admin들이 있을 수 있고

채널 자체도 id를 갖고 있어야 오른쪽 채팅 페이지에 무엇을 보여줄지 식별하기 용이할 것 같습니다.

그리고 채널에 들어있는 유저 정보 자체가 있어야 그 유저의 이름이나 사진 등을 활용할 수 있을 것 같기도 합니다.

```suggestion
export default interface ChannelType {
  id: number;
  name: string;
  isPrivate: boolean;
   operator: number;
  admins: number[]; 
  members: IUserData[];
  size: number;
}
```

Reply:
음 일단 최대한 쓰이는 데이터 위주로만 타이핑을 해놔서 현재 쓰지 않는 항목에 대해서는 따로 정의해두지 않았습니다. 차피 백엔드에서 네이밍이 달라지면 이를 변경해야 되서 더 불편해질것 같은 느낌도 있고 해서요. 추가적으로 필요한 데이터는 추후 정의하는 식으로 남겨두었습니다. 이후 해당 내용이 필요하면 정의할 예정입니다.